### DISCUSSION TITLE
bug

### DISCUSSION THREAD

**Discussion by shiguyi** [qier222/YesPlayMusic#1139] [Q&A]
代码clone在本地开发的时候启动项目出现proxy代理错误是什么问题呐？

**Comment by pan93412** [comment]
有先拷貝 `.env.example` 以及啟動 `netease_api` 嗎？我當初也是被這兩件事情坑到 😂

**Reply by shiguyi** [reply]
.env.example 在项目里，VUE_APP_NETEASE_API_URL=/api
VUE_APP_ELECTRON_API_URL=/api
VUE_APP_ELECTRON_API_URL_DEV=http://127.0.0.1:10754
VUE_APP_LASTFM_API_KEY=09c55292403d961aa517ff7f5e8a3d9c
VUE_APP_LASTFM_API_SHARED_SECRET=307c9fda32b3904e53654baff215cb67
DEV_SERVER_PORT=20201

netease_api要怎么启动吗？

**Comment by shiguyi** [comment]
.env.example 里面我改这样了VUE_APP_NETEASE_API_URL=http://localhost:3000。netease_api启动是什么问题呐

**Reply by pan93412** [reply]
https://github.com/qier222/YesPlayMusic#computer-配置开发环境 這裡的第二個程式碼區塊w
